workers.py

```diff
@@ -156,10 +157,20 @@ def get_pod_spec(self, build):
                                 "cpu": cpu,
                                 "memory": mem
                             }
+                        },
+                        "volumeMounts": {
+                            "mountPath": "/scratch",
```

Comment:
you'll also have to configure pg and mysql to use this dir instead of their default

Reply:
Yes, this is WIP and will be in a separate PR.

Reply:
Database changes are in #102.